### ISSUE TITLE
Git introduction should fork the repository

### ISSUE THREAD

**Issue by willgraf** [vanvalenlab/intro-to-deepcell#15]
Users cannot push to the project if they are not members of the vanvalenlab organization. Instead, they should fork their own repository and make commits to the forked repo. This should remove any authorization issues.

**Comment by ngreenwald** [comment]
I assume this is referring to the intro coding section, correct?

**Comment by willgraf** [comment]
Yeah I think this is specifically for the `git` section, where the tutorial walks through how to submit Pull Requests. I'm not sure how relevant this is.
